benches/storage_cmp.rs

```diff
@@ -87,7 +87,7 @@ where
             let entities = world.entities();
             let storage = world.read_storage::<C>();
 
-            for e in (&*entities).join() {
+            for e in (&entities).join() {
```

Comment:
If I recall correctly, the parenthesis here are also unnecessary, i.e. you can do `for e in entities.join() {}`. I've been able to do that with components, at least, so hopefully that still applies with `Entities` now 🤞